pkg/kubecost/allocation.go

```diff
@@ -2115,6 +2120,21 @@ func deriveProportionalAssetResourceCosts(options *AllocationAggregationOptions,
 			}
 		}
 
+		for name, pvAlloc := range alloc.PVs {
+			// log.Infof("Inserting PARC: %+v", ProportionalAssetResourceCost{
```

Comment:
can toss this

Reply:
Weird, I deleted this, and force pushed, but it's still here in GH :thinking:

Reply:
![Screenshot from 2023-07-28 10-00-15](https://github.com/opencost/opencost/assets/8070055/992c5c8e-7941-4861-b9f3-d5d33d4bdbdd)
![Screenshot from 2023-07-28 10-00-32](https://github.com/opencost/opencost/assets/8070055/06428c2b-ebf1-45c2-81dc-0d4ecf03d506)